libs/components/tsconfig.lib.json

```diff
@@ -1,14 +1,11 @@
 {
     "extends": "./tsconfig.json",
-    "files": ["../../cypress/types.d.ts"],
+    "files": ["../../cypress/types.d.ts", "./src/progress-bar/newFile.ts"],
```

Comment:
Is dit nodig?

Reply:
nee ik heb dat per ongeluk gecreerd. Nu is het verwijderd.